packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -243,6 +248,82 @@ export class CdkToolkit {
     }
   }
 
+  public async watch(options: WatchOptions) {
+    const rootDir = path.dirname(path.resolve(PROJECT_CONFIG));
+    debug("root directory used for 'watch' is: %s", rootDir);
+
+    const watchSettings: { include?: string | string[], exclude: string | string [] } | undefined =
+        this.props.configuration.settings.get(['watch']);
+    if (!watchSettings) {
+      throw new Error("Cannot use the 'watch' command without specifying at least one directory to monitor. " +
+        'Make sure to add a "watch" key to your cdk.json');
+    }
+
+    // For the "include" subkey under the "watch" key, the behavior is:
+    // 1. No "watch" setting? We error out.
+    // 2. "watch" setting without an "include" key? We default to observing "./**".
+    // 3. "watch" setting with an empty "include" key? We default to observing "./**".
```

Comment:
I'm defaulting to observing `./**` in case there is no `"include"` key found in `cdk.json`. I can be convinced that this is trying to be too clever, and we shouldn't do it.

Reply:
I am okay with that